TypeScript definition - Wrong return type for getPixelColor()

<!-- IF YOU DO NOT FILL OUT THE FOLLOWING FORM YOUR ISSUE WILL AUTOMATICALLY BE CLOSED!!! -->

# Expected Behavior

getPixelColor(x, y) should return a number

# Current Behavior

The TypeScript definition says getPixelColor(x, y) returns a Jimp object, causing compilation to fail

# Failure Information (for bugs)

## Steps to Reproduce

Try to compile this typescript code
```
let image: Jimp = await Jimp.read('anypicture.png');
let color: number = image.getPixelColor(1, 1);
```
You'll get an error:
```
error TS2322: Type 'Jimp' is not assignable to type 'number'.
```


**Screenshots**
N/A

## Context

- Jimp Version: 0.5.4
- Operating System: Windows 10 x64
- Node version: 8.12.0 LTS

## Failure Logs
```
lib/Slicer3D.ts:97:13 - error TS2322: Type 'Jimp' is not assignable to type 'number'.

97             color = this.finalImage.getPixelColor(x, j);
```

# Additional information

Fixing compilation was trivial: I replaced:
```
    getPixelColor(
      x: number,
      y: number,
      cb?: GenericCallback<number, any, this>
    ): this;
```
with
```
    getPixelColor(
      x: number,
      y: number,
      cb?: GenericCallback<number, any, this>
    ): number;
```
in `jimp.d.ts`

I'm pretty sure there are a bunch of other functions that shouldn't return "this" but should return number instead (like getPixelColour, getPixelIndex, etc), but I don't use them.

The issue is absent from Jimp 0.2.28, which I've fallen back to for the moment. For instance, [this old version of jimp.d.ts](https://github.com/lygstate/jimp/blob/09b1f4598e33057d4f6ae2fc795dfcf43f0a5b59/jimp.d.ts) did indicate the return value of getPixelColor to be a number.

--- a/packages/jimp/jimp.d.ts
+++ b/packages/jimp/jimp.d.ts
@@ -290,12 +290,12 @@ declare namespace Jimp {
       x: number,
       y: number,
       cb?: GenericCallback<number, any, this>
-    ): this;
+    ): number;
     getPixelColour(
       x: number,
       y: number,
       cb?: GenericCallback<number, any, this>
-    ): this;
+    ): number;
     setPixelColor(
       hex: number,
       x: number,
@@ -553,7 +553,11 @@ declare namespace Jimp {
       cb: Jimp.GenericCallback<Font, any, any>
     ): Promise<never>;
     static measureText(font: Font, text: PrintableText): number;
-    static measureTextHeight(font: Font, text: PrintableText, maxWidth: number): number;
+    static measureTextHeight(
+      font: Font,
+      text: PrintableText,
+      maxWidth: number
+    ): number;
   }
 }
 

